test/sema/dm_option.f90

```diff
@@ -0,0 +1,59 @@
+!
```

Comment:
Technically this is a preprocessor test. Can we create a new sub-directory in test called preprocessor and add this test there? You might need to copy some files like `lit.local.cfg`.

Reply:
test location changed to `test/preprocessor/dm_option.f90`
-->fixed